Tests/Features/Context/VictoireContext.php

```diff
@@ -863,4 +865,63 @@ private function getPage()
 
         return $session->getPage();
     }
+
+    /**
+     * Run the history reminder command.
+     *
+     * @When /^I run the sitemap generation command$/
+     */
+    public function iRunTheSitemapGenerationCommand()
+    {
+        $phpFinder = new PhpExecutableFinder();
+        if (false === $php = $phpFinder->find()) {
+            throw new \RuntimeException('Unable to find the PHP executable.');
+        }
+        $phpBin = $php;
+        $process = new Process(null);
+
+        $process->setWorkingDirectory($this->getContainer()->getParameter('kernel.root_dir').'/..');
```

Comment:
Lines from `$phpFinder = new PhpExecutableFinder();` to `$process->setWorkingDirectory(…);` are present 3 times. It can be moved in a method to avoid code duplication.